### PR TITLE
shades_colorGenerator by Ukesh-dev

### LINKED ISSUE #51: Shades-colorGenerator
# Description :
A website for generating the different shades of colors.
## Features :
- Having the funtionality to copy the selected colors to clipboard. 
- Made using Reactjs.

### PR DESCRIPTION
**Pull request by Ukesh-dev**
### Shades_ColorGenerator

- A website with the functionality to generate and copy the different shades of colors to your clipboard.
- Made with Reactjs.

Closes #51  

## Add GIF/ScreenShot 
shades

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [x] You have created a new branch for contribution.
- [x] Added README.md for your project in a specified folder.
- [x] Add Demo in GIF or Screenshot form.
- [x] You are Registered Officially for HacktoberFest 2021 as Partcipant.
- [x] Followed Contribution Guidelines.

## Issue 
Mention your issue no. #51

### CODE REVIEW

File: Shades-ColorGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Shades-ColorGenerator-
+
+## A website with functionality to generate and copy the different shades of colors.
+
+![](src/shades.PNG)
```
Comment (soumik2012) [inline Shades-ColorGenerator/README.md:5]: - Add a snapshot of your project in  gif,jpg, png 
- write step set up the project 
-  Write Author name in last

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Minnor Change only then good to go 👍🏻

File: Shades-ColorGenerator/README.md (hunk 1 above)
Comment (Ukesh-dev) [reply Shades-ColorGenerator/README.md]: I guess it should look good.

**Review by Ukesh-dev — COMMENTED** [review]
@soumik2012 Is it okay now?

**Review by soumik2012 — APPROVED** [review]
Great LGTM 💯

**Comment by soumik2012** [comment]
Thank you @Ukesh-dev  For contributing @Tenet-Coding

**Pull request merged**